editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -226,6 +229,26 @@ function useStyleFullyVisible(path: TemplatePath): boolean {
   }, 'NavigatorItem useStyleFullyVisible')
 }
 
+export function isProbablySceneFromMetadata(
+  jsxMetadata: ElementInstanceMetadataMap,
+  path: TemplatePath,
+): boolean {
+  const elementMetadata = MetadataUtils.findElementByTemplatePath(jsxMetadata, path)
+  return (
+    elementMetadata != null &&
+    isRight(elementMetadata.element) &&
```

Comment:
Probably be ever so slightly cleaner with a `foldEither`.

Reply:
Yep that's fair. I've taken a note (physically on my notepad in front of me) and will change that when I return to the tidying up work.